homework.md

```diff
@@ -4,110 +4,111 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
-What is the  2 * 3 * 6 * 1 * 10 * 34?
+What is the 2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
-## Question 4 
+## Question 4
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks better.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+Java Script is a dynamic programming language that allows you to make webpages interactive.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1 - Git and Git hub let you work together on the same project with branching capabilities.
+2 - It keeps all of the history of whatever each person has done on the project.
+3 - People from all around the world can work on the same project with Git & Github and they are good tools for collaboration.
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You make a copy of the repository on your GitHup account.
 
-## Question 10 
+## Question 10
 
-What happens when you clone a repostory?
+What happens when you clone a repository?
 
-It send it to a friend
+you bring the repository on your GitDesktop.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
+When you have done all of your changes in the project you will pull it to be reviewed and then merged to the main repository.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It lets you have the history and record of whatever you have changed and done on the project.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+Pushing send and upload the commit history of your local repository to Github.
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+It's a web service to store and share our codes.
 
 ## Question 15
 
 What does it mean to "Commit your changes"?
 
-Send you code to Github
+Send your code to Github with the explanation about what you have changed.
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+Visual Studio Code.
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To inform others what changes you have made to the code.
 
 ## Question 18
 
 Where can I look to find help with Git?
 
-The Git Cheatsheet!
+The Git Cheat sheet!
```

Comment:
Hi Laleh, these are really good answers. You can think of  adding Git documentation for this answer.